app/addons/documents/views.js

```diff
@@ -453,25 +458,31 @@ function(app, FauxtonAPI, Components, Documents, Databases, Views, QueryOptions)
       }
 
       this.toggleTrash();
+      this.updateExpandButtonLabel();
     },
 
     serialize: function() {
       return {
         viewList: this.viewList,
-        expandDocs: this.expandDocs,
         endOfResults: !this.pagination.canShowNextfn()
       };
     },
 
-    collapse: function (event) {
-      event.preventDefault();
+    toggleExpandCollapse: function (e) {
+      e.preventDefault();
+      var expand = this.$(e.target).find("i").hasClass('icon-plus');
 
-      if (this.expandDocs) {
-        this.expandDocs = false;
```

Comment:
this will get changed back again once the headerbar in react is merged - we can't use the dom for storing state there (which is a good thing) and depend on `this.expandDocs`

is there a special reason to switch to the usage of classes? We can decide to use the `this.expandDocs` here again or we will have to change that when one of us is having a huge merge conflict and maybe there are reasons to use the class that would introduce a bug using the old `this.expandDocs`